unyt/_unit_lookup_table.py

```diff
@@ -195,7 +200,11 @@
             ),
         ),
         ("cal", (4.184, dimensions.energy, 0.0, r"\rm{cal}", True)),
-        ("BTU", (1055.0559, dimensions.energy, 0.0, r"\rm{BTU}", False)),
+        ("BTU", (J_per_BTU, dimensions.energy, 0.0, r"\rm{BTU}", False)),
+        ("MMBTU", (btu_per_mmbtu, dimensions.energy, 0.0, r"\rm{MMBTU}", False)),
+        ("therm", (btu_per_therm, dimensions.energy, 0.0, r"\rm{therm}", False)),
+        ("quad", (btu_per_quad, dimensions.energy, 0.0, r"\rm{quad}", False)),
+        ("Wh", (watt_per_hour, dimensions.energy, 0.0, r"\rm{Wh}", True)),
```

Comment:
According to wikipedia, allowing SI prefixes for this one is also a bit controversial. In the interest of practicality I think it's fine.